fares-data-build-tool/fdbt-site
# Added secondary input validation for definePassengerType page

# Description

-   This PR fixes the validation issues on TFN-453 which allowed a user to input a minimum age greater than the maximum age (and vice versa).

# Testing instructions

-   Pull down this branch and run the site locally.
-   Navigate to the definePassengerType page and attempt to enter a maximum age less than the minimum age, and vice versa.
-   Check that the errors displayed for other error scenarios are still showing as expected and that the page still functions as previously on a valid input.

# Type of change


# Checklist:
